crates/valence_new/examples/resource_pack.rs

```diff
@@ -71,7 +71,7 @@ fn prompt_on_punch(mut clients: Query<&mut Client>, mut events: EventReader<Inte
                 "https://github.com/valence-rs/valence/raw/main/assets/example_pack.zip",
                 "d7c6108849fb190ec2a49f2d38b7f1f897d9ce9f",
                 false,
-                None,
+                Option::<Text>::None,
```

Comment:
Can we revert this back to just `None`?

Reply:
There is an `Option<impl Into<Text>>` now, so it could be reversed if the function is changed back as well. Should I?

Reply:
Yes, thank you.